pkg/metricsservice/api/metrics.pb.go

```diff
@@ -15,8 +15,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.32.0
```

Comment:
Could you help to double check this one for me ? Seems like we are dowgrading the version :thinking: